src/main/java/org/osiam/resources/validation/ValidCoreSchema.java

```diff
@@ -0,0 +1,26 @@
+package org.osiam.resources.validation;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.*;
+
+@Target({ElementType.METHOD, ElementType.FIELD, ElementType.ANNOTATION_TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = CoreSchemaValidator.class)
+@Documented
+public @interface ValidCoreSchema {
+    String message() default "Invalid core schema";
+
+    Class<?>[] groups() default {};
+
+    Class<? extends Payload>[] payload() default {};
+
+    String value();
+
+    @Target({ElementType.METHOD, ElementType.FIELD, ElementType.ANNOTATION_TYPE})
+    @Retention(RetentionPolicy.RUNTIME)
+    @Documented
+    @interface List {
```

Comment:
That's right, it's not needed anywhere, but good practice when writing validators, because it allows the application of lists in the validator. That said: I wouldn't be unhappy when I removed it. Do you think it has to go?